server/sonar-server/src/main/java/org/sonar/server/notification/ws/RemoveAction.java

```diff
@@ -100,8 +97,8 @@ public void define(WebService.NewController context) {
         "  <li>Global notifications: %s</li>" +
         "  <li>Per project notifications: %s</li>" +
         "</ul>",
-        globalDispatchers.stream().sorted().collect(Collectors.joining(", ")),
-        projectDispatchers.stream().sorted().collect(Collectors.joining(", ")))
+        dispatchers.getSortedGlobalDispatchers().stream().sorted().collect(Collectors.joining(", ")),
```

Comment:
minor: I'd rename to `getGlobalDispatchers()`

Reply:
In fact it was my first intention but then I think that it's important to say that the list is sorted.